parse/process_options.go

```diff
@@ -21,3 +22,7 @@ func WithWriter(w io.Writer) OptionsFunc {
 func WithDebug() OptionsFunc {
 	return func(o *options) { o.debug = true }
 }
+
+func WithEvents(c chan<- *Event) OptionsFunc {
```

Comment:
This should probably be an `Event` value, instead of a pointer, as we don't want anything outside the `parse` package to mutate the struct, wdyt?

Reply:
Sure thing, done!